encoding/per-scene-adaptation/index.html

```diff
@@ -1,7 +1,7 @@
 <script type="text/javascript"
         src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
 <script type="text/javascript"
-        src="https://cdn.bitmovin.com/analytics/web/beta/2/bitmovinanalytics.min.js"></script>
+        src="https://cdn.bitmovin.com/analytics/web/2.5.2/bitmovinanalytics.min.js"></script>
```

Comment:
Please don’t use a specific version here or we’ll need to update the demos all the time